### ISSUE TITLE
Dynamic Prices

### ISSUE THREAD

**Issue by divios** [divios/DailyShop#15]
Remodel the entire prices system and implement a good stable complex system, with dynamic prices, that changes the price according to the demand or more parameters

**Comment by MidnightSugar** [comment]
I don't think you should totally remove the random pricing system, instead maybe have dynamic pricing be only an option.

**Comment by divios** [comment]
Yeah you are right, I still have to think about how should I implement it but is going to be a complex feature. Keeping the dynamic prices is a good idea since is a more friendly feature

**Comment by Igzak** [comment]
Any updates for this feature?
